### PR TITLE
ZWave Panel Updates

### PR DESCRIPTION
**Pull request by cgarwood**
3 updates:
- Added padding to the OZW log section so it's not right against the card border
- The "Entities of this node" dropdown now shows the actual entity_id instead of a half-entity_id half-friendly variation.
- Removed the "Node Information" card and replaced it with a "Node Information" button that opens the more-info popup for the zwave node. This allows for renaming from the zwave panel using the pre-existing cog in the more-info dialog, and gets rid of duplicate code, since the original Node Information card only showed the attributes of the zwave node, like the more-info card already does.

### CODE REVIEW

**Comment by cgarwood** [comment]
Screenshot with the new Node Information button and entity_id in the "Entities of this node" dropdown:
image

**Comment by turbokongen** [comment]
I like it! Very good change 👍

File: src/panels/config/zwave/ha-config-zwave.js [hunk 1]
```diff
@@ -563,6 +557,11 @@ class HaConfigZwave extends LocalizeMixin(PolymerElement) {
     };
   }
 
+  nodeMoreInfo(ev) {
```
Comment (balloob) [inline src/panels/config/zwave/ha-config-zwave.js]: Can you prefix private methods with `_`

File: src/panels/config/zwave/ha-config-zwave.js [hunk 2]
```diff
@@ -563,6 +557,11 @@ class HaConfigZwave extends LocalizeMixin(PolymerElement) {
...
+    ev.preventDefault();
```
Comment (balloob) [inline src/panels/config/zwave/ha-config-zwave.js]: I don't think that this is necessary?

File: src/panels/config/zwave/ha-config-zwave.js [hunk 3]
```diff
@@ -563,6 +557,11 @@ class HaConfigZwave extends LocalizeMixin(PolymerElement) {
...
+    this.fire('hass-more-info', { entityId: this.nodes[this.selectedNode].entity_id });
```
Comment (balloob) [inline src/panels/config/zwave/ha-config-zwave.js]: You'll need to include the EventsMixin to get `this.fire`

File: src/panels/config/zwave/ha-config-zwave.js (hunk 2 above)
Comment (cgarwood) [reply src/panels/config/zwave/ha-config-zwave.js]: I originally copied from the more-info code in dev-states, but tested it without this line and it looks like it still works, so I'll remove it.

**Comment by cgarwood** [comment]
Should be good to review again.

**Review by balloob — APPROVED** [review]

**Pull request merged**